packages/rpc-core/src/rpc-methods/common.ts

```diff
@@ -29,6 +29,11 @@ export type Slot = U64UnsafeBeyond2Pow53Minus1;
 // the JSON-RPC transport.
 export type U64UnsafeBeyond2Pow53Minus1 = bigint;
 
+// FIXME(solana-labs/solana/issues/30341) Beware that any value outside of range
+// +/- 9007199254740991 may be truncated or rounded because of a downcast to JavaScript `number`
+// between your calling code and the JSON-RPC transport.
+export type LamportsPlusOrMinusAsI64Unsafe = bigint;
```

Comment:
Maybe this?
```suggestion
export type SignedLamportsAsI64Unsafe = bigint;
```

Too much overloading the term ‘signed’?